Refine release notes for 8.5 (#26608)
gradle/gradle: subprojects/docs/src/docs/release/notes.md

}
```

> 

Author: ```suggestion
```

}
```

> This was done to ensure that version catalogs are accessible via API in situations where generated accessors are not.

Author: @eskatos, I updated the statement but it still doesn't feel very impactful.

Author: I'm not a fan of this, I will delete and merge.

Reviewer: The new statement is more accurate. Feel free to omit though, I think it just paraphrases what's above.

> This was done to ensure that version catalogs are accessible via API in situations where generated accessors are not.

Author: ```suggestion
```

> This was done to ensure that version catalogs are accessible during the configuration phase of a multi-module project, avoiding cumbersome workarounds.

Author: @eskatos, is this an accurate statement?

Author: I want to make sure release notes don't just address the **what** but also the **why**. Why did we add this? How does it benefit our users?

> This was done to ensure that version catalogs are accessible during the configuration phase of a multi-module project, avoiding cumbersome workarounds.

Author: ```suggestion
This was done to ensure that version catalogs are accessible via API in situations where generated accessors are not.
```
